src/imitation/scripts/train_adversarial.py

```diff
@@ -53,8 +55,9 @@ def train_adversarial(
     checkpoint_interval: int,
     gen_batch_size: int,
     init_rl_kwargs: Mapping,
+    reward_net_cls: Optional[Type[reward_nets.RewardNet]],
```

Comment:
These new arguments are missing from the docstring

Reply:
Oh, good catch, I was relying on darglint to fix this but it looks like it's not running on CI -- I'll fix that in a separate PR. Added these arguments in now.